ballsdex/packages/players/cog.py

```diff
@@ -143,6 +151,30 @@ async def mention_policy(self, interaction: discord.Interaction, policy: Mention
             f"Your mention policy has been set to **{policy.name.lower()}**.", ephemeral=True
         )
 
+    @friend.command(name="policy")
+    @app_commands.choices(
+        policy=[
+            app_commands.Choice(name="Accept all friend requests", value=MentionPolicy.ALLOW),
+            app_commands.Choice(name="Deny all friend requests", value=MentionPolicy.DENY),
```

Comment:
This needs to be changed as it uses the same as mentions.
```suggestion
            app_commands.Choice(name="Accept all friend requests", value=FriendPolicy.ALLOW),
            app_commands.Choice(name="Deny all friend requests", value=FriendPolicy.DENY),
```

Reply:
dumb mistake on my part sorry